src/plugins/curator/components/gallery-fields/index.jsx

```diff
@@ -92,6 +92,7 @@ const GalleryFields = ({ isEditMode, gallery, formName, nodeRef, schemas }) => {
           name="allowComments"
         />
         <hr />
+        <Label style={{ display: 'block' }}>Related Galleries</Label>
```

Comment:
@cohencodes - why add here when you can just pass a `label` prop to the `GalleryPickerField`? see here: https://github.com/triniti/cms-js/blob/master/src/plugins/curator/components/gallery-picker-field/index.jsx#L24

update the code in the `picker-field` to show the label in both view & edit modes.

Reply:
@arunkarnati it has two labels, one inside ("SEARCH AND SELECT GALLERIES") and one outside ("RELATED GALLERIES")